test/unit/gateways/redsys_rest_test.rb

```diff
@@ -0,0 +1,264 @@
+require 'test_helper'
+
+class RedsysRestTest < Test::Unit::TestCase
+  include CommStub
+
+  def setup
+    @gateway = RedsysRestGateway.new(fixtures(:redsys))
```

Comment:
Can you update this to be `redsys_rest` and add mock credentials to the `fixtures.yml` file? It is currently pulling the old `redsys` credentials but the key length of the secret is too short causing the tests to fail.

Reply:
I think maybe a while back Redsys switched to issuing longer secret keys for all accounts.